rust/src/proofs/api.rs

```diff
@@ -1009,7 +1009,7 @@ pub unsafe extern "C" fn fil_get_post_version(
     registered_post_proof_accessor(registered_proof, |p| Ok(format!("{:?}", p)))
 }
 
-unsafe fn registered_seal_proof_accessor(
+pub unsafe fn registered_seal_proof_accessor(
```

Comment:
Question: What is the intent of making these public? Do you wish to expose them to Go?

Reply:
There were warnings about them, no other intent.